src/codegen/mod.rs

```diff
@@ -1007,8 +1011,10 @@ impl CodeGenerator for CompInfo {
             };
 
             let mut attrs = vec![];
-            if let Some(comment) = field.comment() {
-                attrs.push(attributes::doc(comment));
+            if ctx.options().generate_comments {
```

Comment:
That's right, but I thought that the alternative would be less readable? Anyway, will file.